src/popup/components/TransferReview.vue

```diff
@@ -79,6 +82,10 @@
         />
       </template>
     </DetailsItem>
+    <PayloadDetail
```

Comment:
Please add empty line before and after this component same as you did with other components.